src/components/views/rooms/EventTile.js

```diff
@@ -486,6 +492,14 @@ module.exports = withMatrixClient(React.createClass({
         return this.props.getRelationsForEvent(eventId, "m.annotation", "m.reaction");
     },
 
+    _onReactionsCreated(relationType, eventType) {
+        if (relationType !== "m.annotation" || eventType !== "m.reaction") {
+            return;
+        }
+        this.props.mxEvent.removeListener("Event.relationsCreated", this._onReactionsCreated);
+        this.forceUpdate();
```

Comment:
comment for why would be nice, as we should be avoiding this.

Reply:
I think I was just being a bit lazy here at the time... I've [changed](https://github.com/matrix-org/matrix-react-sdk/pull/2954/commits/af9fdbaeebd0dff2465a1a4f66f9bb985a8c5a62) this to use state instead.